### PR TITLE
Merge pull request #132 from Project-MONAI/setup_readthedocs

### PR DESCRIPTION
**Pull request by vikashg**
Setup ReadTheDocs

### CODE REVIEW

**Comment by MMelQin** [comment]
I believe a rebase will help since [PR #132](https://github.com/Project-MONAI/monai-deploy-app-sdk/pull/132) has been merged.

File: docs/source/introduction/roadmap.md [hunk 1]
```diff
@@ -1 +1,5 @@
 # Roadmap
+
+**v0.1.0** is focusing on implementing MVP to give you a sense of what MONAI Deploy App SDK's API would look like, focusing on the disk input/output.
+
```
Comment (gigony) [inline docs/source/introduction/roadmap.md:4]: test

**Review by MMelQin — APPROVED** [review]
Looks good.

**Review by MMelQin — APPROVED** [review]
Looks good.

**Comment by gigony** [comment]
@vikashg Is it okay to close this?

**Pull request closed without merging**